final_prep/mod_zero_hero.rb

```diff
@@ -1,42 +1,62 @@
 # Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 # Declare two variables - hero_name AND special_ability - set to strings
+hero_name = 'Frosty'
+special_ability = 'make everything touched lukewarm'
 
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
+greeting = "Hello, my name is #{hero_name}!"
+catchphrase = "I can #{special_ability}!"
 
 # Declare two variables - power AND energy - set to integers
+power = 3
+energy = 10
 
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
+full_power = (power * 500)
+full_energy = (energy + 150)
 
-# Declare two variables - is_human and identity_concealed - assigned to booleans
 
+# Declare two variables - is_human and identity_concealed - assigned to booleans
+is_human = false
+identity_concealed = true
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
+arch_enemies = ['Steam Roller', 'Hot Tip', 'The Insulator']
+sidekicks = ['Frosted Flake', 'Mini Me', 'Carrot Top']
 
 # Print the first sidekick to your terminal
-
+p sidekicks[0]
 # Print the last arch_enemy to the terminal
-
+p arch_enemies[2]
```

Comment:
How would you print the last `arch_enemy` if you didn't know the length of the array?